composer.json

```diff
@@ -247,7 +247,8 @@
         }
     },
     "bin": [
-        "manager-bundle/bin/contao-console"
+        "manager-bundle/bin/contao-console",
+        "manager-bundle/bin/contao-script-handler"
```

Comment:
I really don't like that name, it does not say anything about what that is for. `contao-loader` or `contao-initializer` or `contao-setup` or something thelike. Remember it can be called outside composer, right?

Reply:
Agreed, I don't like `contao-script-handler` either.

@all What's your favored name? What about `contao-auto-scripts` to play along Sf flex? Or is this too magic sounding? :slightly_smiling_face:

Reply:
I like `contao-setup`.